internal/pkg/levee/testdata/src/levee_analysistest/example/tests/callcommon/tests.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package callcommon contains tests for Defer and Go instructions,
+// which should be treated similarly to Call instructions.
+package callcommon
+
+import (
+	"fmt"
+	"io"
+	"levee_analysistest/example/core"
+)
+
+func TestGoBuiltin(dst, src []interface{}, s core.Source) {
+	src[0] = s
+	go copy(dst, src)
+	core.Sink(dst) // want "a source has reached a sink"
+}
+
+func TestDeferBuiltin(dst, src []interface{}, s core.Source) {
+	src[0] = s
+	defer copy(dst, src)
+	core.Sink(dst) // want "a source has reached a sink"
+}
```

Comment:
This isn't expected to actually propagate.  Please add a ` // TODO (opened issue number)`.

Reply:
[Outdated.  Resolving.]